web/src/components/templates/Template.svelte

```diff
@@ -146,13 +147,25 @@
     Qualifications uses Worker Qualifications defined on MTurk.
   </div>
 
-  <div class="max-w-xs">
-    <Label forID="selectionType" text="Participant Selection Type" />
-    <Select
-      id="selectionType"
-      bind:value={template.selectionType}
-      options={selectionTypes}
-      placeholder="Particpant Selection Type" />
+  <div class="md:grid grid-cols-3 gap-2">
+    <div class="max-w-xs">
+      <Label forID="selectionType" text="Participant Selection Type" />
+      <Select
+        id="selectionType"
+        bind:value={template.selectionType}
+        options={selectionTypes}
+        placeholder="Particpant Selection Type" />
+    </div>
+    {#if template.selectionType === 'INTERNAL_DB'}
+      <div class="flex flex-column pt-5 max-w-xs">
+        <Checkbox bind:checked={template.internalCriteria.uninitialized}>
+          <Label
+            forID="unitializedParticipants"
+            text="Select Uninitialized Participants"
+            question="Uninitialized Participants are Participants that have been imported, and do not yet have a HIT associated with them, so we cannot send messages or bonuses to them" />
```

Comment:
I guess given my previous comment, we should add some details here. At the end of the description, you can add: 

"Uninitialized participants are *not* included in an Internal DB selection. On the contrary, if you check this box, *only* uninitialized participants will be selected."

Hum, that's a bit of a mouthful... Hopefully at least it's clear.